lib/prop/limiter.rb

```diff
@@ -46,8 +46,7 @@ def before_throttle(&blk)
       #
       # Raises Prop::RateLimited if the number if the threshold for this handle has been reached
       def configure(handle, defaults)
-        raise ArgumentError.new("Invalid threshold setting") unless defaults[:threshold].to_i > 0
-        raise ArgumentError.new("Invalid interval setting")  unless defaults[:interval].to_i > 0
+        Prop::Options.build(key: nil, params: {}, defaults: defaults)
```

Comment:
check that this does not cause side-effects / extra work ... prefer a static "validate_options" method
```suggestion
        Prop::Options.build(key: nil, params: {}, defaults: defaults) # validate options
```